### DISCUSSION TITLE
Feature Request: 瀑布流视图，在设置里，是否可以自定义列数

### DISCUSSION THREAD

**Discussion by lj723911** [Quorafind/Obsidian-Thino#579] [Polls 投票]
### Feature Requested | 请求的功能

瀑布流视图目前固定是三列，这个状态会同步到移动端，展示不友好。哪怕是PC端，3列也不能满足特定场景下的需求。

### Checklist | 检查清单

- [X] The feature would be useful to more users than just me. 该功能对于不仅仅是我自己的用户来说是有用的。

[a comment by an automated account is not shown]

**Comment by juestchaos** [comment]
建议由此需求的用户都讨论下
自定义宽度功能会对用户屏幕或者ob 使用习惯的宽度有要求，可能遇到一些边界和极限情况
会引发适配和，展示样式的问题

**Reply by lj723911** [reply]
不是自定义宽度，而是有更多列数可以选择。简单来做，目前默认是3，那是否可以支持2~5，每列还是等宽的。这样的方案我觉得并不会比现在强制3列有更多的适配问题。

[a comment by an automated account is not shown]

**Comment by Quorafind** [comment]
移动端情况下应当是默认单列视图，但是如果你开启了页面宽度不够时也允许展示别的视图的话，会导致你的视图并不可见。我的建议是在两端验证过以后，不要同步 data.json 文件（尤其是当前的宽度问题导致的需求下）

[a comment by an automated account is not shown]
